src/templates/index-page.js

```diff
@@ -1,17 +1,53 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { graphql } from 'gatsby';
+import { Link, graphql } from 'gatsby';
 import Layout from '../components/Layout';
+import styled from 'styled-components';
 import Content, { HTMLContent } from '../components/Content';
+import InfographicPage from '../pages/infographic-page';
+import { bpMaxMD } from '../lib/breakpoints';
+
+const HeroQuestion = styled.div`
+  color: rgb(14, 61, 119);
+  font-size: 2.5em;
+  font-family: 'Oswald', sans-serif;
+  font-weight: bold;
+  text-align: right;
+  padding: 4px;
+  min-width: 340px;
+  margin: 0 0 1em;
+  ${bpMaxMD} {
+    display: none;
+  }
+`;
 
 export const IndexPageTemplate = ({ title, content, contentComponent }) => {
   const PageContent = contentComponent || Content;
 
   return (
     <section className="section section--gradient">
+      <div className="centered">
+        <HeroQuestion>I want to...</HeroQuestion>
+        <div className="big-button">
+          <Link to="/request-supplies">Volunteer</Link>
```

Comment:
This link should be:
```suggestion
          <Link to="/volunteer">Volunteer</Link>
```
shouldn't it?

Reply:
Yes! Thank you! Will change now!